packages/vm/src/instance.rs

```diff
@@ -107,6 +109,12 @@ where
                 "humanize_address" => Func::new(move |ctx: &mut Ctx, canonical_ptr: u32, human_ptr: u32| -> i32 {
                     do_human_address(api, ctx, canonical_ptr, human_ptr)
                 }),
+                "humanize_address" => Func::new(move |ctx: &mut Ctx, canonical_ptr: u32, human_ptr: u32| -> i32 {
+                    do_human_address(api, ctx, canonical_ptr, human_ptr)
+                }),
```

Comment:
The humanize_address was duplicated. Merge error?

Reply:
Yup